cmd/cerbos/compile/internal/verification/internal/diff.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021-2022 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package internal
+
+import (
+	"fmt"
+
+	"github.com/cerbos/cerbos/cmd/cerbos/compile/internal/colored"
+)
+
+func PrintDiff(diff map[string]string) (string, error) {
```

Comment:
Wouldn't it be easier to use a struct to hold the expected and actual values?  :-)

```go
type Outcome struct {
    Expected effectv1.Effect `json:"expected"`
    Actual effectv1.Effect `json:"actual"`
}

func (o Outcome) Display() string {
    return fmt.Sprintf("Expected: %s Actual: %s", colored.FailedTest(expected), colored.FailedTest(actual))
}
```